src/leaflet_property.js

```diff
@@ -55,3 +55,33 @@ export const clusterIconNumberStyles = `
   color: white;
```

Comment:
Would it make sense to call this leaflet_properties?

Also, would it be better for organizations if it was under a `constants` folder?